tests/migration/eviction_strategy.go

```diff
@@ -292,11 +304,16 @@ var _ = SIGMigrationDescribe("Live Migration", func() {
 						cfg.MigrationConfiguration.NodeDrainTaintKey = &drain
 						tests.UpdateKubeVirtConfigValueAndWait(cfg)
 					}
-					setControlPlaneSchedulability(false)
+
+					controlPlaneNodes := libnode.GetControlPlaneNodes(virtClient)
+
+					// Make sure the vmi will not be scheduled in the control-plane node in those clusters where there is only one.
+					unschedulableControlPlaneNode = controlPlaneNodes.Items[0].Name
+					libnode.SetNodeUnschedulable(unschedulableControlPlaneNode, virtClient)
```

Comment:
This is something I was thinking to but honestly I was afraid that will result in some flakyness, since the prefferred option can and cannot be respected. WDYT?